packages/client/utils/taskStatus.ts

```diff
@@ -10,7 +11,7 @@ export const taskStatusLabels: Record<TaskStatus, TaskStatusLabel> = {
   [TaskStatus.PRIVATE]: TaskStatusLabel.PRIVATE
 } as const
 
-export const taskStatusColors: Record<TaskStatus, string> = {
+export const taskStatusColors: Record<TaskStatus, ValueOf<typeof PALETTE>> = {
```

Comment:
this is a pretty tame example, but it's showing that we have a string union here, not just `string`.